adoptopenjdk-api-v3-updater/src/test/kotlin/net/adoptopenjdk/api/ReleaseVersionResolverTest.kt

```diff
@@ -82,7 +82,7 @@ class ReleaseVersionResolverTest : BaseTest() {
     @Test
     fun mostRecentFeatureVersionIsCorrect() {
         check { releaseInfo ->
-            releaseInfo.most_recent_feature_version == 13
+            releaseInfo.most_recent_feature_version == 12
```

Comment:
This may be confusing to the layperson if they think 15 should be the latest?

Reply:
this is based on test assets so should not change over time